platform/avr-ravenusb/cdc_task.c

```diff
@@ -75,6 +75,10 @@
 #include <avr/wdt.h>
 #include <util/delay.h>
 
+#if UIP_CONF_IPV6_RPL
+#include "rpl.h"
+#endif
+
```

Comment:
Why is this needed up here? Maybe add a comment, otherwise someone might move it down again.